eng/tools/summarize-impact/src/impact.ts

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env node
 
-import * as fs from "fs";
+import { existsSync, readFileSync } from "fs";
 import { glob } from "glob";
-import * as path from "path";
+import { dirname, join, resolve } from "path";
```

Comment:
We may not be completely consistent, but we usually import only the specific APIs we need.